target/linux/rockchip/patches-5.10/106-rockchip-rk3328-OpenWrt-LED-tweak-for-OrangePi-R1-Plus.patch

```diff
@@ -0,0 +1,16 @@
+--- a/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus.dts
++++ b/arch/arm64/boot/dts/rockchip/rk3328-orangepi-r1-plus.dts
+@@ -13,6 +13,13 @@
+ 	model = "Xunlong Orange Pi R1 Plus";
+ 	compatible = "xunlong,orangepi-r1-plus", "rockchip,rk3328";
+ 
++	aliases {
```

Comment:
Why do you add this as a separate patch and don't include it in the original patch?